srtcore/channel.h

```diff
@@ -264,30 +257,33 @@ class CChannel
 
         if (adr.family() == AF_INET)
         {
-            mh.msg_control = m_acCmsgSendBuffer;
-            mh.msg_controllen = CMSG_SPACE(sizeof(in_pktinfo));
-            cmsghdr* cmsg_send = CMSG_FIRSTHDR(&mh);
+            mh.msg_control = (void *) buf;
+            mh.msg_controllen = sizeof(struct in_pktinfo) + sizeof(cmsghdr);
 
+            cmsghdr* cmsg_send = CMSG_FIRSTHDR(&mh);
             // after initializing msghdr & control data to CMSG_SPACE(sizeof(struct in_pktinfo))
```

Comment:
I think you can just as well remove this comment, especially that it's actually not true.